doc/changelog.rst

```diff
@@ -1,6 +1,23 @@
 Changelog
 =========
 
+Changes in Version 4.4
+----------------------
+
+Version 4.4 brings a number of improvements including:
+
+- Added support for :ref:`CSFLE on-demand credentials` for cloud KMS providers.
```

Comment:
I think this can go into the 4.3.3 release if we market it as "documented support" instead of "added support". We can say that it requires pymongocrypt>=1.4. I think this makes more sense because it will be supported by older versions of pymongo too.

Reply:
Sounds good